lib/mobile/main.asm

```diff
@@ -6545,11 +6545,11 @@ Function112b11:
 	ld [wc872], a
 	ret
 
-Function112b60:
+.asm_112b60:
 	ld a, [hli]
-	cp $30
+	cp "0"
 	jr c, .asm_112b6f
-	cp $3a
+	cp "9" + 1
```

Comment:
```suggestion
	cp ":"
```

Reply:
> Curious, why the char + 1's?

Its checking for ascii characters between "0" and "9" inclusive, "9" + 1 makes it easier to see that than comparing against ":"

Reply:
> > Curious, why the char + 1's?
> 
> Its checking for ascii characters between "0" and "9" inclusive, "9" + 1 makes it easier to see that than comparing against ":"

That makes a Sense! Thanks for the explanation!